source/features/parse-backticks.tsx

```diff
@@ -25,6 +25,9 @@ function init(): void {
 		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalDiscussionList`
 		'[data-channel^="check_suites"] a', // `isActions`
 		'.repository-content .pr-toolbar h2', // `isActions` run
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden a:first-child', // "Repository refresh" layout commit title
```

Comment:
@kidonng can you follow the comment style. We add by each selector what page this selector is used on. `isIssue` etc.

BTW thanks for all your PR's!